magda-registry-api/README.md

```diff
@@ -51,7 +51,7 @@ users in different organization units. See the comment in the sql file for detai
 ### Start opa server
 
 ```
-cd deploy/helm/magda-core/charts/opa
+cd deploy/helm/magda/magda-core/charts/opa
```

Comment:
probably should be `deploy/helm/magda-core/charts/opa` (without /magda/).

Reply:
double oops 😬